screed/screedRecord.py

```diff
@@ -129,6 +112,18 @@ def __ne__(self, given):
         except AttributeError:
             raise TypeError("Cannot compare to given type: %s" % type(given))
 
+    def __lt__(self):
+        return NotImplementedError
```

Comment:
https://docs.python.org/2/library/stdtypes.html?highlight=notimplemented#the-notimplemented-object